examples/scale/backend/backend.go

```diff
@@ -150,33 +152,49 @@ func runFetchMatches(be pb.BackendServiceClient, p *pb.MatchProfile, matchesForA
 	}
 }
 
-func runAssignments(be pb.BackendServiceClient, matchesForAssignment <-chan *pb.Match, ticketsForDeletion chan<- string) {
+func runAssignments(be pb.BackendServiceClient, matchesForAssignment <-chan *pb.Match, ticketsForDeletion chan<- string, backfillsForAcknowledgment chan<- string) {
 	ctx := context.Background()
 
 	for m := range matchesForAssignment {
 		ids := []string{}
 		for _, t := range m.Tickets {
 			ids = append(ids, t.GetId())
 		}
-
+		logger.WithFields(logrus.Fields{
+			"Backfill":      m.Backfill,
+			"TicketsNumber": len(m.Tickets),
+		}).Info("Received a match with Backfill")
+
+		if m.Backfill != nil {
+			logger.WithFields(logrus.Fields{
+				"matchID":       m.MatchId,
+				"BackfillID":    m.Backfill.Id,
+				"TicketsNumber": len(m.Tickets),
+			}).Info("Received a match with Backfill ID")
+		}
```

Comment:
why not to put the upper log, the one with "Received a match with Backfill", inside an else statement from this if??